deployment/nic-configuration-operator-chart/templates/operator.yaml

```diff
@@ -59,3 +59,12 @@ spec:
             periodSeconds: 10
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - name: firmware-cache
+              mountPath: /nic-firmware
+              readOnly: false
+      # TODO change to user-specified storage class
+      volumes:
```

Comment:
Just to make sure I understand, eventually we use a PVC with shared storage (ReadWriteMany)?
Where the PVC name will be configured?

Reply:
Eventually we are going to require users to provide a storage class, that would allow the operator and node agents access to the same shared storage. The operator is going to have rw access and the node agents are going to have read-only permissions.

Reply:
Updated the PR to use the user-provided storage class for NIC FW provisioning